### ISSUE TITLE
Accessibility for RenderWhiteSpace broken

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#1097]
Issue Type: <b>Bug</b>

Currently Render white space is either drawing or putting something in the place of spaces that screen readers cannot speak or render to blind and low vision users.

Expected behavior. When RenderWhiteSpace is turned to all the following could be used to replace the space characters so that Screen readers will speak them. This works in other editors by doing this. Note that this is just an example of what could be used. If you think better symbols would work that is fine if both jaws and NVDA speak the characters. The following characters have been evaluated and work.

Space – 0x2024 says <dot>
End of line – 0x2193 says <downwards arrow>
Tab - 0x2192 says <rightwards arrow> or maybe 2023 says <arrow bullet>

It would be nice if the RenderWhiteSpace is pre-defined as a key press to toggle on and off to what ever the setting is set to. This way a blind person could press it and check the indentation and quickly press it to turn off the visible spaces. If the renderWhiteSpace is able to be bound by a user, I guess the default key press does not have to be defined. but it would be nice.

VS Code version: Code 1.67.2 (c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5, 2022-05-17T18:15:52.058Z)
OS version: Windows_NT x64 10.0.19044
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7820HQ CPU - 2.90GHz (8 x 2904)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.88GB (23.52GB free)|
|Process Argv|--crash-reporter-id ea0256fe-448a-4761-b67e-49aad19149df|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (6)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-intelephense-client|bme|1.8.2
python|ms-|2022.6.3
vscode-pylance|ms-|2022.5.3
jupyter|ms-|2022.4.1021342353
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.8

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyl392:30443607
pythontb:30283811
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
vscscmwlcmt:30465135
cppdebug:30492333
vsclangdc:30486549

```

</details>

We have written the needed data into your clipboard because it was too large to send. Please paste.
